pyproject.toml

```diff
@@ -132,7 +135,38 @@ add-ignore = ['D107', 'D407', 'D417']
 
 [tool.setuptools]
 include-package-data = true
+license-files = ['LICENSE']
 
 [tool.setuptools.packages.find]
 include = ['copulas', 'copulas.*']
 namespaces = false
+
+[tool.bumpversion]
+current_version = "0.10.1.dev0"
```

Comment:
So this means our version will live in the following places:
- copulas/__init__.py
- pyproject.toml

Will bump-my-version handle updating it in both places? Based on the code below, it seems like the answer is Yes but just double checking.

Reply:
Yeah exactly, I confirmed that both get updated when running `bump-my-version bump`